app/models/user.rb

```diff
@@ -7,6 +7,7 @@ class User < ActiveRecord::Base
   validates :email, presence: true, length: {maximum: 255},
             format: {with: VALID_EMAIL_REGEX},
             uniqueness: {case_sensitive: false}
+  validates :school, presence: true
```

Comment:
this validation would belong on the student model class, not the user class.  THis class corresponds to rows in the user table.